docs/docsite/rst/using/installing.rst

```diff
@@ -240,25 +240,52 @@ the Ansible doc site <https://docs.ansible.com/ansible/latest/user_guide/playboo
 
 .. _installing_multi_repos:
 
-Multi-role Repositories
+Collections
 =======================
 
-Traditionally, a role is a git repository. Galaxy v3.0 introduced multi-role repositories, providing the
-ability to combine multiple roles into a single git repository.
-
-Installing a mult-role repository requires using the ``mazer`` command line tool available at the `Ansible
+Installing collections requires using the ``mazer`` command line tool available at the `Ansible
 Mazer project <https://github.com/ansible/mazer>`_.
 
 .. note::
 
-  This is a tech-preview feature. Future releases of Galaxy may change or break support of multi-role
-  repositories.
+  This is a tech-preview feature. Future releases of Galaxy may change or break support of collections.
+
+Collections are installed using mazer's ``install`` command, and passing the *namespace.collection_name*,
+as demonstrated by the following command:
+
+.. code-block:: bash
+
+  $ mazer install namespace.collection_name
 
-All the roles from a multi-role repository can be installed using mazer's ``install`` command, and passing the
-*namespace.repository_name*, as demonstrated by the following command:
+Older versions of collections can also be installed by using
```

Comment:
Does "also" mean you can install two versions side by side or just that you can pick which one version to install?

Reply:
It's not possible to do side by side installs of multiple versions of the same collection in a single 'collections path' (ie, ~/.ansible/collections).

It is possible to specify a specific version (or require a version spec) from the cli and from collection dependencies. (At the moment, `mazer install alikins.collection_whatever,version=1.2.3` which is kind of awkward, but https://github.com/ansible/mazer/issues/24 is related).

And it is possible to install the same collection to two different collection paths (~/.ansible/collections user path and /usr/share/ansible/collections/ system paths for ex).
Each collection path can have a different version installed. ansible/ansible-playbook will search for the collection namespace.name in each of the paths defined in ANSIBLE_COLLECTION_PATHS _in order_ and use the first one found.

Reply:
We should definitely point out that only a single version of a Collection can be installed. What happens if I have version '1.0.1' installed, and I ask to install a prior version, '1.0.0'? Or, what happens if I have version '1.0.1' installed, and I ask to install new version '1.0.2'? It would be nice to know how these cases are handled by Mazer.